index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="assets/style/style.css">
+  <title>Cinema | Showcase</title>
+</head>
+<body>
+  <header class="header-bar">
+    <nav class="nav-bar">
+      <img class="humberger" src="assets/icons/menu.png" alt="menu" id="menubtn">
+      <ul class="menu" id="menu">
+        <div class="close-menu" id="closeMenu"></div>
+        <li><a href="index.html#">Home</a></li>
+        <li><a href="#programSection">Programs</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#newEvents">News</a></li>
+        <li><a href="#topMovies">Movies</a></li>
+        <li><a href="#ourPartners">Our partners</a></li>
+        <li class="pres"><button type="button" class="prettion">presentation</button></li>
+      </ul>
+    </nav>
+    <div class="logo">
+      <img src="assets/icons/cinema.png" alt="This is a cinema logo">
+    </div>
+  </header>
+   <main class="main">
+      <section class="welcome-events">
+        <div class="presnt">
+          <div class="welcome">
+            <p class="wlcm">Welcome to Cinema Showcase</p>
+            <div class="presentation">
+              <video class="vid" autoplay muted loop src="assets/img/wave.mp4"></video>
+              <p>Unlock Ultimate Movie Magic!</p>
+            </div>
+          </div>
+          <p class="welcome-paragraph">
+            Welcome to Cinema Showcase! Get ready to be entertained with classic and contemporary films, movie reviews, interviews, and more!
+          </p>
+        </div>
+        <div class="cine-events" id="newEvents">
+          
+        </div>
+      </section>
+      <section class="main-program" id="programSection">
+        <div class="programs-title">
+          <h3>Main Program</h3>
+          <hr>
+        </div>
+        <ul class="programs-list" id="allPrograms">
```

Comment:
- [ ] Kindly add the `hover effect` to your cards to match the original design

![image](https://user-images.githubusercontent.com/98453979/225988099-57a7346e-cc3a-40e9-ad6e-afbf8a6369e6.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/225988363-4168d342-f3ad-4e31-9f93-f0cf9fbbf278.png)

Reply:
Done